proto/serverless/instrumentation/tags/v1/notice.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package serverless.instrumentation.tags.v1;
+
+option go_package = ".;protoc";
+
+message NoticeTags {
+    // Notice Message
+    string message = 1;
```

Comment:
If we are going to have a single type here, can we get it written out what possible messages will be here given the shortened set of NoticeType's?

Reply:
@max Do you mean to list here all the messages we configure in code (like `Binary body was removed`) and ensure this list is updated whenever we update the grammar of these messages or add new ones?

Reply:
Ok, I think you rather mean listing fingerprints (message codes), as we should never rely on messages programmatically (they're either not used, or human readable messages for users)